### PR TITLE
merged singleton-deployment using profile

### PR DESCRIPTION
**Pull request by tommaso-borgato**

### CODE REVIEW

File: clusterbench-ee7-ear/pom.xml [hunk 1]
```diff
@@ -432,6 +432,68 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>singleton-deployment-v1</id>
+            <build>
+                <finalName>clusterbench-ee7-singleton-deployment-v1</finalName>
```
Comment (rhusar) [inline clusterbench-ee7-ear/pom.xml]: I don't find the v1 vs v2 descriptive. Why not call them by what's the modification in each?

Also, this sounds more like functional testing aspects, i.e. just a different way to annotate deployments as singleton. Do we really need both here?

Comment (tommaso-borgato) [reply clusterbench-ee7-ear/pom.xml]: changed v1 and v2 with some more descriptive names! I would like to keep it here to avoid spreading info .....

**Comment by rhusar** [comment]
@tommaso-borgato Fails CI.

**Comment by tommaso-borgato** [comment]
> @tommaso-borgato Fails CI.

Yes, but it fails on deployment of clusterbench-ee8.ear that was not touched in this PR

**Pull request merged**